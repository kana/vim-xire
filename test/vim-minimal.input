execute 'source'
\       substitute(expand('<sfile>'), '\([^/\\]*\)\.input$', ',\1.vim', '')

" vim: filetype=vim
