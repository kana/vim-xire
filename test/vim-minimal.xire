; Minimal xire script - nothing is generated from this source.
;
; vim: filetype=scheme
