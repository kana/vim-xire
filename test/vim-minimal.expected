


**** Result ****


0 examples, 0 failures
